src/utils/utils.tsx

```diff
@@ -0,0 +1,10 @@
+export const isEven = (n: number) => {
+  if (n === 0) {
+    return true
+  }
+  if ((n ^ 1) === n + 1) {
```

Comment:
can be done in one if condition
```suggestion
  if ((n ^ 1) === n + 1 || n === 0) {
  return true
  }
  return false
```

Reply:
Done